File: week1/pset1/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int height = 0;
    // Request the pyramid height from the user
    while (height < 1 || height > 8)
    {
        height = get_int("Height: ");
    }
    // Making a pyramid
    for (int width = 1; width <= height; width++)
    {
        // Making a empty symbols
        for (int n = 0; n < height - width; n++)
        {
            printf(" ");
        }
        // Making a # symbols
        for (int j = 0; j < width; j++)
        {
            printf("#");
        }
        // New string and - 1
        printf("\n");
    }
}

File: week6/pset6/sentimental-mario-less/mario.py
# Import library cs50
from cs50 import get_int


# Request the pyramid height from the user
while True:
    height = get_int("Height: ")
    if height > 0 and height < 9:
        break
# Loop for mario blocks
width = height - 1
for i in range(1, height + 1):
    print(" " * width, end="")
    print("#" * i, end="")
    width -= 1
    print("")